ui/client/config.ts

```diff
@@ -0,0 +1,19 @@
+/*
+  We want to be able to serve NK under different (in particular not empty) paths (e.g. via nginx configuration) without rebuilding frontends
+  The easiest way would be to use relative urls, but because we don't use hash router, but rely on different paths, it's more difficult
+  Currently we replace __publicPath__ string in main.html during serving with configured path, and we use "auto" publicPath from webpack
+  via global variable (__webpack_public_path__).
+  __webpack_public_path__ is also used when this module is imported in external app (WebpackModuleFederationPlugin) because it contains
```

Comment:
Look like important entry to documentation about how to use federated modules

Reply:
if you use Webpack, you know where it's docs lays. same with WebpackModuleFederationPlugin.  I think its enough.